BothamNetworking/BothamAPIClient.swift

```diff
@@ -134,4 +143,8 @@ public class BothamAPIClient {
         return httpClient.hasValidScheme(request)
 
     }
+}
+
+private struct BothamAPIClientErrorCode {
+    private static let networkError = -1009
```

Comment:
Take a look at the following page, there is constant for theses values, that have more explicit names.

http://nshipster.com/nserror/
https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Miscellaneous/Foundation_Constants/index.html#//apple_ref/doc/constant_group/URL_Loading_System_Error_Codes